lib/coarse_op.cuh

```diff
@@ -624,58 +172,287 @@ namespace quda {
       default:
 	errorQuda("Undefined compute type %d", type);
       }
+
       return threads;
     }
 
     bool tuneGridDim() const { return false; } // don't tune the grid dimension
     bool tuneAuxDim() const { return type != COMPUTE_VUV ? false : true; }
 
     unsigned int sharedBytesPerBlock(const TuneParam &param) const {
-      if (arg.shared_atomic && type == COMPUTE_VUV)
+      if (/*arg.shared_atomic && */type == COMPUTE_VUV)
```

Comment:
Can we get rid of the `arg.shared_atomic`? Otherwise add a comment on why it's still there for reference

Reply:
I've deleted the commented out `arg.shared_atomic`.  We now no longer template on global vs shared memory atomic versions of `computeVUV`, and it is a run-time option that we auto tune over.  So we now always have to declare this shared memory requirement.